### ISSUE TITLE
Training set loss goes down, validation set loss and mpjpe goes up 

### ISSUE THREAD

**Issue by psile** [tamasino52/UGCN#4]
Hi, I meet a problem when python main.py --train, that is training set loss goes down, validation set loss and mpjpe goes up, I think it is abnormal because the loss and mpjpe of validation set goes up in the begining.

The validation result when training even worse than the Initialization model.

Thank you for your reply if possible.

**Comment by liuyangme** [comment]
Hi! I encountered the same problem. Have you solved it? 
Thank you for your reply.
